app/models/book.rb

```diff
@@ -12,4 +12,8 @@ class Book < ApplicationRecord
 
   scope :active, -> { where(active: true) }
   scope :inactive, -> { where(active: false) }
+
+  def copies_available
+    return self.max_copies - Loan.on_loan.where(book: self).count
```

Comment:
Build is failing because we don't have a presence validation of books.max_copies. This means that a book can exist in the database WITHOUT specifying the number of max copies.

To solve this:

1. Add a presence validation for max_copies in the Book model
2. Ensure that our specs and fixtures specify a `max_copies` value for all Book records